### ISSUE TITLE
Forgotten closing </recipe> Lines in SAGMillRecipes_USER.xml

### ISSUE THREAD

**Issue by chefkooch** [CaiganMythFang/TheFerretBusinessIssues#53]
Hi everybody,
I've noticed that the closing </recipe> statements are missing in the Sagmill recipes for Silver Trade Chit, Silver Shop Chip and Silver bla bla bla.

**Sympthoms in the game:**
Sagmill recipes for Ferret Business items do not work

**Sympthoms in Minecraft STDOUT:**
```
[08:01:24] [Client thread/WARN] [Railcraft]: Could not register CraftGuide plugins: CraftGuide missing.
[08:01:25] [Client thread/INFO] [OpenComputers]: Done with post init phase.
[08:01:25] [Client thread/INFO] [STDOUT]: [mcp.mobius.waila.gui.truetyper.TrueTypeFont:<init>:98]: TrueTypeFont loaded: java.awt.Font[family=Minecraftia,name=Minecraftia Regular,style=plain,size=14] - AntiAlias = true
[08:01:26] [Client thread/WARN] [EnderIO]: RecipeParser: A new recipe was started before the recipe was closed.
[08:01:26] [Client thread/WARN] [EnderIO]: RecipeParser: A new recipe was started before the recipe was closed.
[08:01:26] [Client thread/WARN] [EnderIO]: RecipeParser: A new recipe was started before the recipe was closed.
[Fatal Error] :3108:9: The end-tag for element type "recipe" must end with a '>' delimiter.
```

**System Info:**
I'm playing the Modpack on a Mac OS Sierra via the Twitch Launcher and Updated to the 0.2.5 Version of the Pack.

I hope that helps. Other than that I just want to say great Modpack guys! Keep up the good work.
Best regards

Bob

**Comment by CaiganMythFang** [comment]
Yeah, I made one set, tested it and it worked, then accidentally flubbed one of the recipe lines out tabbing through windows and copied it over for the other two sets and didn't test it because I was silly.

This is already fixed on my end, will be in the next update which I'm planning to be a small-ish one with some quests and a bunch of bug fixes.

**Comment by CaiganMythFang** [comment]
Fixed in 0.2.6
